src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -50,7 +50,7 @@ public EditCommand parse(String args) throws ParseException {
         }
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_NRIC, PREFIX_PHONE, PREFIX_EMAIL,
-                PREFIX_ADDRESS, PREFIX_APPOINTMENT, PREFIX_MEDICAL, PREFIX_TAG);
+                PREFIX_ADDRESS, PREFIX_APPOINTMENT);
```

Comment:
This is to allow the program to proceed even if multiple t/ or m/ tags are detected